Godeps/Godeps.json

```diff
@@ -1171,127 +1171,127 @@
 		},
 		{
 			"ImportPath": "github.com/gogo/protobuf/gogoproto",
-			"Comment": "v0.4-3-gc0656edd",
+			"Comment": "v0.4-3-gc0656ed",
```

Comment:
There is an actual release - https://github.com/gogo/protobuf/releases/tag/v0.4
I hope a similar release would be available for other dependencies. 
I'd like to avoid vendoring in HEAD of dependencies.
@wojtek-t @shyamjvs since you are experts in this area, can you weigh in?